msgraph-mail/typescript/src/users/item/inferenceClassification/overrides/overridesRequestBuilder.ts

```diff
@@ -83,11 +85,11 @@ export class OverridesRequestBuilder {
      * @param responseHandler Response handler to use in place of the default response handling provided by the core service
      * @returns a Promise of InferenceClassificationOverride
      */
-    public post(body: InferenceClassificationOverride | undefined, requestConfiguration?: OverridesRequestBuilderPostRequestConfiguration | undefined, responseHandler?: ResponseHandler | undefined) : Promise<InferenceClassificationOverride | undefined> {
+    public post(body: InferenceClassificationOverride | undefined, requestConfiguration?: OverridesRequestBuilderPostRequestConfiguration | undefined, responseHandler?: ResponseHandler | undefined) : Promise<InferenceClassificationOverrideImpl | undefined> {
```

Comment:
executor methods that accept a request body that's a complex type need to new up the complex type with the JSON value as a parameter so we get all the serialization information back before passing it to the request adapter.